force-app/volunteers/reports/VolunteerReports/VolunteerHoursByMonth.report-meta.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
It looks like the matrix grouping for month is missing. If we add that should we get rid of Total Volunteer Hours by Month or come up with a way to disambiguate?